src/main/java/bio/terra/service/dataset/flight/upgrade/enableSecureMonitoring/EnableSecureMonitoringCreateSourceDatasetAndSnapshotsTpsPolicyStep.java

```diff
@@ -41,7 +38,10 @@ public EnableSecureMonitoringCreateSourceDatasetAndSnapshotsTpsPolicyStep(
 
   @Override
   public StepResult doStep(FlightContext context) throws InterruptedException {
-    List<UUID> snapshotsToCreatePolicies = enumerateSnapshotIdsForDataset(context);
+    FlightMap workingMap = context.getWorkingMap();
+    UUID datasetId = workingMap.get(DatasetWorkingMapKeys.DATASET_ID, UUID.class);
+    List<UUID> snapshotsToCreatePolicies =
```

Comment:
same thing about the variable naming...maybe policiesToCreate? Your call

Reply:
Hmm these are the snapshot ids to create policies for (and those ids are also the policy ids) 🤔 I just simplified it to `snapshotIds`, which I think is okay in the context of this step!

Reply:
Actually...I changed my mind 😅 (naming is hard)!

Reply:
lol sorry.  Also, I totally agree about naming being impossible